system/Common.php

```diff
@@ -437,7 +438,7 @@ function esc($data, string $context = 'html', ?string $encoding = null)
             // Provide a way to NOT escape data since
             // this could be called automatically by
             // the View library.
-            if (empty($context) || $context === 'raw') {
+            if ($context === 'raw') {
```

Comment:
please revert, '0' is empty, ref https://3v4l.org/mgSdd

this PR should only change PHPDoc, change code may cause regression.

Reply:
If it needs to be applied, I think it should go to 4.3 as it changed behaviour.

Reply:
@samsonasik Technically you are correct.

But I cannot imagine users who write code like `esc($string, '0')`.
I see no problem with this change in develop.

Reply:
Ok then 👍

Reply:
@samsonasik I think accepting `esc($string, '0')` is a bug.
I'll fix it in another PR.
Thank you for commenting!

Reply:
I sent #6722